2024-08-08 20:14:58,407 INFO    StreamThr :351347 [internal.py:wandb_internal():86] W&B internal server running at pid: 351347, started at: 2024-08-08 20:14:58.407692
2024-08-08 20:14:58,408 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: status
2024-08-08 20:14:58,409 INFO    WriterThread:351347 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/run-kc0q8dwv.wandb
2024-08-08 20:14:58,409 DEBUG   SenderThread:351347 [sender.py:send():382] send: header
2024-08-08 20:14:58,446 DEBUG   SenderThread:351347 [sender.py:send():382] send: run
2024-08-08 20:14:58,703 INFO    SenderThread:351347 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files
2024-08-08 20:14:58,703 INFO    SenderThread:351347 [sender.py:_start_run_threads():1136] run started: kc0q8dwv with start time 1723119298.407659
2024-08-08 20:14:58,718 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: check_version
2024-08-08 20:14:58,718 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: check_version
2024-08-08 20:14:58,842 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: run_start
2024-08-08 20:14:58,852 DEBUG   HandlerThread:351347 [system_info.py:__init__():32] System info init
2024-08-08 20:14:58,852 DEBUG   HandlerThread:351347 [system_info.py:__init__():47] System info init done
2024-08-08 20:14:58,852 INFO    HandlerThread:351347 [system_monitor.py:start():194] Starting system monitor
2024-08-08 20:14:58,852 INFO    SystemMonitor:351347 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-08 20:14:58,852 INFO    SystemMonitor:351347 [interfaces.py:start():190] Started cpu monitoring
2024-08-08 20:14:58,852 INFO    HandlerThread:351347 [system_monitor.py:probe():214] Collecting system info
2024-08-08 20:14:58,853 INFO    SystemMonitor:351347 [interfaces.py:start():190] Started disk monitoring
2024-08-08 20:14:58,854 INFO    SystemMonitor:351347 [interfaces.py:start():190] Started gpu monitoring
2024-08-08 20:14:58,854 INFO    SystemMonitor:351347 [interfaces.py:start():190] Started memory monitoring
2024-08-08 20:14:58,854 INFO    SystemMonitor:351347 [interfaces.py:start():190] Started network monitoring
2024-08-08 20:14:58,863 DEBUG   HandlerThread:351347 [system_info.py:probe():196] Probing system
2024-08-08 20:14:58,865 DEBUG   HandlerThread:351347 [system_info.py:_probe_git():181] Probing git
2024-08-08 20:14:58,871 DEBUG   HandlerThread:351347 [system_info.py:_probe_git():189] Probing git done
2024-08-08 20:14:58,871 DEBUG   HandlerThread:351347 [system_info.py:probe():244] Probing system done
2024-08-08 20:14:58,871 DEBUG   HandlerThread:351347 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-44-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-08T12:14:58.863762', 'startedAt': '2024-08-08T12:14:58.403207', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/qwen2-1.5b', '--use_lora', 'True', '--init_lora_weights', 'pissa_niter_4', '--target_modules', 'q_proj,v_proj,k_proj,o_proj,gate_proj,down_proj,up_proj', '--lora_rank', '32', '--lora_alpha', '32', '--lora_dropout', '0', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:100000]', '--output_dir', '/home/rwkv/JL/out_model/llama-test', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '1', '--gradient_accumulation_steps', '32', '--save_strategy', 'steps', '--save_steps', '1000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': '74dce7f004b5b16bfd24c8ca22e737aee6ff0769'}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 1207.4011428571428, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 5496.371, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 2096.56, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 4949.581, 'min': 800.0, 'max': 5500.0}, {'current': 5499.156, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5081.87, 'min': 800.0, 'max': 5600.0}, {'current': 5436.664, 'min': 800.0, 'max': 5600.0}, {'current': 3466.294, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 5499.079, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 5092.223, 'min': 800.0, 'max': 5500.0}, {'current': 4299.185, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 481.3404006958008}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.55397415161133}}
2024-08-08 20:14:58,871 INFO    HandlerThread:351347 [system_monitor.py:probe():224] Finished collecting system info
2024-08-08 20:14:58,871 INFO    HandlerThread:351347 [system_monitor.py:probe():227] Publishing system info
2024-08-08 20:14:58,871 DEBUG   HandlerThread:351347 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-08 20:14:58,871 DEBUG   HandlerThread:351347 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-08 20:14:58,871 DEBUG   HandlerThread:351347 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-08 20:14:59,704 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/conda-environment.yaml
2024-08-08 20:14:59,704 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/requirements.txt
2024-08-08 20:15:00,547 DEBUG   HandlerThread:351347 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-08 20:15:00,547 DEBUG   HandlerThread:351347 [system_info.py:_save_code():90] Saving code
2024-08-08 20:15:00,549 DEBUG   HandlerThread:351347 [system_info.py:_save_code():111] Saving code done
2024-08-08 20:15:00,549 DEBUG   HandlerThread:351347 [system_info.py:_save_patches():128] Saving git patches
2024-08-08 20:15:00,587 DEBUG   HandlerThread:351347 [system_info.py:_save_patches():170] Saving git patches done
2024-08-08 20:15:00,587 INFO    HandlerThread:351347 [system_monitor.py:probe():229] Finished publishing system info
2024-08-08 20:15:00,589 DEBUG   SenderThread:351347 [sender.py:send():382] send: files
2024-08-08 20:15:00,589 INFO    SenderThread:351347 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-08 20:15:00,589 INFO    SenderThread:351347 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-08 20:15:00,590 INFO    SenderThread:351347 [sender.py:_save_file():1392] saving file diff.patch with policy now
2024-08-08 20:15:00,595 DEBUG   SenderThread:351347 [sender.py:send():382] send: telemetry
2024-08-08 20:15:00,640 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: stop_status
2024-08-08 20:15:00,640 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-08 20:15:00,641 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: stop_status
2024-08-08 20:15:00,703 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/conda-environment.yaml
2024-08-08 20:15:00,703 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/code/llama/gmm.py
2024-08-08 20:15:00,703 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/diff.patch
2024-08-08 20:15:00,703 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/wandb-metadata.json
2024-08-08 20:15:00,704 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/code/llama
2024-08-08 20:15:00,704 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/code
2024-08-08 20:15:00,977 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9
2024-08-08 20:15:00,977 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:00,977 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9BFDFEB7E70D6</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:00,990 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb
2024-08-08 20:15:00,991 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5514', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:00,991 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9BFDFEA04D301</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:01,259 DEBUG   SenderThread:351347 [sender.py:send():382] send: config
2024-08-08 20:15:01,260 DEBUG   SenderThread:351347 [sender.py:send():382] send: metric
2024-08-08 20:15:01,261 DEBUG   SenderThread:351347 [sender.py:send():382] send: telemetry
2024-08-08 20:15:01,261 DEBUG   SenderThread:351347 [sender.py:send():382] send: metric
2024-08-08 20:15:01,261 WARNING SenderThread:351347 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-08 20:15:01,261 DEBUG   SenderThread:351347 [sender.py:send():382] send: telemetry
2024-08-08 20:15:01,261 DEBUG   SenderThread:351347 [sender.py:send():382] send: telemetry
2024-08-08 20:15:01,261 DEBUG   SenderThread:351347 [sender.py:send():382] send: config
2024-08-08 20:15:01,264 DEBUG   SenderThread:351347 [sender.py:send():382] send: exit
2024-08-08 20:15:01,265 INFO    SenderThread:351347 [sender.py:send_exit():589] handling exit code: 1
2024-08-08 20:15:01,265 INFO    SenderThread:351347 [sender.py:send_exit():591] handling runtime: 2
2024-08-08 20:15:01,267 INFO    SenderThread:351347 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-08 20:15:01,267 INFO    SenderThread:351347 [sender.py:send_exit():597] send defer
2024-08-08 20:15:01,268 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: defer
2024-08-08 20:15:01,268 INFO    HandlerThread:351347 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-08 20:15:01,269 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: defer
2024-08-08 20:15:01,269 INFO    SenderThread:351347 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-08 20:15:01,269 INFO    SenderThread:351347 [sender.py:transition_state():617] send defer: 1
2024-08-08 20:15:01,269 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: defer
2024-08-08 20:15:01,269 INFO    HandlerThread:351347 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-08 20:15:01,270 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: defer
2024-08-08 20:15:01,270 INFO    SenderThread:351347 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-08 20:15:01,270 INFO    SenderThread:351347 [sender.py:transition_state():617] send defer: 2
2024-08-08 20:15:01,270 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: defer
2024-08-08 20:15:01,271 INFO    HandlerThread:351347 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-08 20:15:01,271 INFO    HandlerThread:351347 [system_monitor.py:finish():203] Stopping system monitor
2024-08-08 20:15:01,271 DEBUG   SystemMonitor:351347 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-08 20:15:01,271 DEBUG   SystemMonitor:351347 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-08 20:15:01,271 DEBUG   SystemMonitor:351347 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-08 20:15:01,272 INFO    HandlerThread:351347 [interfaces.py:finish():202] Joined cpu monitor
2024-08-08 20:15:01,273 INFO    HandlerThread:351347 [interfaces.py:finish():202] Joined disk monitor
2024-08-08 20:15:01,297 INFO    HandlerThread:351347 [interfaces.py:finish():202] Joined gpu monitor
2024-08-08 20:15:01,297 INFO    HandlerThread:351347 [interfaces.py:finish():202] Joined memory monitor
2024-08-08 20:15:01,297 INFO    HandlerThread:351347 [interfaces.py:finish():202] Joined network monitor
2024-08-08 20:15:01,297 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: defer
2024-08-08 20:15:01,298 INFO    SenderThread:351347 [sender.py:send_request_defer():613] handle sender defer: 2
2024-08-08 20:15:01,298 INFO    SenderThread:351347 [sender.py:transition_state():617] send defer: 3
2024-08-08 20:15:01,298 DEBUG   SenderThread:351347 [sender.py:send():382] send: stats
2024-08-08 20:15:01,298 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: defer
2024-08-08 20:15:01,298 INFO    HandlerThread:351347 [handler.py:handle_request_defer():172] handle defer: 3
2024-08-08 20:15:01,298 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: defer
2024-08-08 20:15:01,298 INFO    SenderThread:351347 [sender.py:send_request_defer():613] handle sender defer: 3
2024-08-08 20:15:01,298 INFO    SenderThread:351347 [sender.py:transition_state():617] send defer: 4
2024-08-08 20:15:01,298 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: defer
2024-08-08 20:15:01,298 INFO    HandlerThread:351347 [handler.py:handle_request_defer():172] handle defer: 4
2024-08-08 20:15:01,298 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: defer
2024-08-08 20:15:01,298 INFO    SenderThread:351347 [sender.py:send_request_defer():613] handle sender defer: 4
2024-08-08 20:15:01,298 INFO    SenderThread:351347 [sender.py:transition_state():617] send defer: 5
2024-08-08 20:15:01,298 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: defer
2024-08-08 20:15:01,298 INFO    HandlerThread:351347 [handler.py:handle_request_defer():172] handle defer: 5
2024-08-08 20:15:01,298 DEBUG   SenderThread:351347 [sender.py:send():382] send: summary
2024-08-08 20:15:01,298 INFO    SenderThread:351347 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-08 20:15:01,298 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: defer
2024-08-08 20:15:01,298 INFO    SenderThread:351347 [sender.py:send_request_defer():613] handle sender defer: 5
2024-08-08 20:15:01,298 INFO    SenderThread:351347 [sender.py:transition_state():617] send defer: 6
2024-08-08 20:15:01,298 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: defer
2024-08-08 20:15:01,298 INFO    HandlerThread:351347 [handler.py:handle_request_defer():172] handle defer: 6
2024-08-08 20:15:01,298 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: defer
2024-08-08 20:15:01,298 INFO    SenderThread:351347 [sender.py:send_request_defer():613] handle sender defer: 6
2024-08-08 20:15:01,300 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: status_report
2024-08-08 20:15:01,477 INFO    SenderThread:351347 [sender.py:transition_state():617] send defer: 7
2024-08-08 20:15:01,477 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: defer
2024-08-08 20:15:01,477 INFO    HandlerThread:351347 [handler.py:handle_request_defer():172] handle defer: 7
2024-08-08 20:15:01,477 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: defer
2024-08-08 20:15:01,477 INFO    SenderThread:351347 [sender.py:send_request_defer():613] handle sender defer: 7
2024-08-08 20:15:01,704 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/config.yaml
2024-08-08 20:15:01,705 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/output.log
2024-08-08 20:15:01,705 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/wandb-summary.json
2024-08-08 20:15:01,815 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68
2024-08-08 20:15:01,816 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '293288', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:01,816 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9BFDFFEC0A0BE</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:01,840 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: poll_exit
2024-08-08 20:15:02,276 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9
2024-08-08 20:15:02,276 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:02,276 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9BFE03B09D1DB</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:02,276 INFO    wandb-upload_1:351347 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-08 20:15:02,307 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb
2024-08-08 20:15:02,307 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5514', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:02,307 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9BFE0344AED5C</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:02,307 INFO    wandb-upload_0:351347 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-08 20:15:03,276 INFO    SenderThread:351347 [sender.py:transition_state():617] send defer: 8
2024-08-08 20:15:03,277 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: poll_exit
2024-08-08 20:15:03,277 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: defer
2024-08-08 20:15:03,277 INFO    HandlerThread:351347 [handler.py:handle_request_defer():172] handle defer: 8
2024-08-08 20:15:03,277 DEBUG   SenderThread:351347 [sender.py:send_request():409] send_request: defer
2024-08-08 20:15:03,277 INFO    SenderThread:351347 [sender.py:send_request_defer():613] handle sender defer: 8
2024-08-08 20:15:03,278 INFO    SenderThread:351347 [job_builder.py:build():298] Attempting to build job artifact
2024-08-08 20:15:03,278 INFO    SenderThread:351347 [job_builder.py:_get_source_type():428] is repo sourced job
2024-08-08 20:15:03,292 INFO    SenderThread:351347 [job_builder.py:build():404] adding wandb-job metadata file
2024-08-08 20:15:03,304 INFO    SenderThread:351347 [sender.py:transition_state():617] send defer: 9
2024-08-08 20:15:03,304 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: defer
2024-08-08 20:15:03,304 INFO    HandlerThread:351347 [handler.py:handle_request_defer():172] handle defer: 9
2024-08-08 20:15:03,305 DEBUG   SenderThread:351347 [sender.py:send():382] send: artifact
2024-08-08 20:15:03,359 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68
2024-08-08 20:15:03,359 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '293288', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:03,359 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9BFE06B644997</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:03,705 INFO    Thread-12 :351347 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240808_201458-kc0q8dwv/files/output.log
2024-08-08 20:15:03,925 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e
2024-08-08 20:15:03,926 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:03,926 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9BFE09E5C0186</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:04,128 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114
2024-08-08 20:15:04,128 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-MD5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-Type': 'application/json', 'Content-Length': '31000', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:04,128 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Resource><Region>wandb-local</Region><RequestId>17E9BFE0A1C290D7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:04,655 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9
2024-08-08 20:15:04,655 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:04,655 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9BFE0C8EC9DB3</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:04,687 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb
2024-08-08 20:15:04,687 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5514', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:04,687 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9BFE0CBF885E3</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:05,165 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e
2024-08-08 20:15:05,165 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:05,165 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9BFE0E83307EB</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:05,508 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114
2024-08-08 20:15:05,508 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-MD5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-Type': 'application/json', 'Content-Length': '31000', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:05,508 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Resource><Region>wandb-local</Region><RequestId>17E9BFE0F3D23FC5</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:06,009 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68
2024-08-08 20:15:06,009 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '293288', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:06,009 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9BFE10C92D2F2</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:06,843 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:15:07,360 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e
2024-08-08 20:15:07,360 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:07,360 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9BFE16AD2B3EE</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:07,360 INFO    wandb-upload_3:351347 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-08 20:15:07,715 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114
2024-08-08 20:15:07,715 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-MD5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-Type': 'application/json', 'Content-Length': '31000', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:07,715 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Resource><Region>wandb-local</Region><RequestId>17E9BFE17F655A86</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:07,715 INFO    wandb-upload_4:351347 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-08 20:15:08,846 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9
2024-08-08 20:15:08,846 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:08,846 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9BFE1C3893284</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:09,168 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb
2024-08-08 20:15:09,168 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5514', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:09,168 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9BFE1D6FF233F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:10,648 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68
2024-08-08 20:15:10,648 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '293288', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:10,649 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9BFE221D08A99</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:11,845 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:15:12,405 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e
2024-08-08 20:15:12,406 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:12,406 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9BFE297993E34</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:12,566 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114
2024-08-08 20:15:12,566 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-MD5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-Type': 'application/json', 'Content-Length': '31000', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:12,566 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Resource><Region>wandb-local</Region><RequestId>17E9BFE296EA5214</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:16,847 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:15:17,918 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb
2024-08-08 20:15:17,918 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5514', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:17,918 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9BFE3E081E8CC</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:18,900 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9
2024-08-08 20:15:18,900 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:18,900 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9BFE41574C016</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:20,531 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68
2024-08-08 20:15:20,531 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '293288', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:20,531 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9BFE4723E5AE6</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:20,583 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e
2024-08-08 20:15:20,583 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:20,583 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9BFE47F041900</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:21,680 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114
2024-08-08 20:15:21,680 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-MD5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-Type': 'application/json', 'Content-Length': '31000', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:21,680 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Resource><Region>wandb-local</Region><RequestId>17E9BFE4B9BF2F90</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:21,848 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:15:26,850 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:15:31,852 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:15:36,187 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=791ea44bc31a453b9d937ba809b6124e573368be8db4f433bac8efa84fdddebb
2024-08-08 20:15:36,187 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5514', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:36,188 ERROR   wandb-upload_0:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E9BFE8216E0649</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:36,854 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:15:38,423 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/diff.patch?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121501Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=80f373eacdab76ac1bf34a96f95adb85a0eaa0d20c5f61a1ed7287c32a0cca68
2024-08-08 20:15:38,424 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '293288', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:38,424 ERROR   wandb-upload_2:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/diff.patch</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/diff.patch</Resource><Region>wandb-local</Region><RequestId>17E9BFE8846BF5FA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:38,626 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=e2664cd7ada212bc9914d954121baeaadc36094675937b114a94304f3adadc6e
2024-08-08 20:15:38,626 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-MD5': 'wt+PCmy04FImuFe2zcU/TQ==', 'Content-Type': 'text/plain; charset=utf-8', 'Content-Length': '5025', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:38,626 ERROR   wandb-upload_3:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/c2df8f0a6cb4e05226b857b6cdc53f4d</Resource><Region>wandb-local</Region><RequestId>17E9BFE8B2C1A343</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:38,648 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121500Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=90a5427332437531ff24cd9253ec63d13e17f351826db937114e4fa3559ebfc9
2024-08-08 20:15:38,648 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14196', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:38,648 ERROR   wandb-upload_1:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/kc0q8dwv/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E9BFE8AFA72484</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:39,106 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240808%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240808T121503Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=content-md5%3Bcontent-type%3Bhost%3Bx-amz-meta-md5&X-User=mobius&X-Amz-Signature=d107356ac0be96be356df1b4efc422047d678293e90c9669e2578b368b83a114
2024-08-08 20:15:39,106 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'x-amz-meta-md5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-MD5': 'Qwrpday/9V8M98yB8ILMVg==', 'Content-Type': 'application/json', 'Content-Length': '31000', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-08 20:15:39,106 ERROR   wandb-upload_4:351347 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Key><BucketName>local-files</BucketName><Resource>/local-files/wandb_artifacts/42/533/430ae975acbff55f0cf7cc81f082cc56</Resource><Region>wandb-local</Region><RequestId>17E9BFE8CB446B1A</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-08 20:15:41,856 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:15:46,858 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:15:51,860 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:15:56,862 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
2024-08-08 20:16:01,863 DEBUG   HandlerThread:351347 [handler.py:handle_request():146] handle_request: keepalive
